docs/containers-transports.5.md

```diff
@@ -11,7 +11,7 @@ containers-transports - description of supported transports for copying and stor
 Tools which use the containers/image library, including skopeo(1), buildah(1), podman(1), all share a common syntax for referring to container images in various locations.
 The general form of the syntax is _transport:details_, where details are dependent on the specified transport, which are documented below.
 
-### **containers-storage:** [storage-specifier]{image-id|docker-reference[@image-id]}
+### **containers-storage**{:storage-specifier} {image-id|docker-reference[@image-id]}
```

Comment:
I think something like
```suggestion
### **containers-storage:**[[**storage-specifier**]**]{image-id|docker-reference[@image-id]}
```
would be accurate (I haven’t checked whether it renders correctly, neither in Markdown nor in the man page).

Reply:
This shows both brackets and everything highlighted.  I've pushed that out, please let me know if further adjustments are necessary.

Reply:
Yeah, sadly the `###` format causes a `.SS` directive, which is bold by default. We could make the replacements italic, but a regular non-bold seems might not be possible to do with Markdown.

Oh well, I suppose we at least got rid of the space.